CHANGES.rst

```diff
@@ -268,9 +212,42 @@ Contributor-facing changes
 
 
 
+Removals and backward incompatible breaking changes
```

Comment:
@bdraco I'd expect breaking changes to be closer to the beginning of the change log… Our current Towncrier config places it right after the deprecations: https://github.com/aio-libs/aiohttp/blob/a561fa9/pyproject.toml#L43-L48.

Any reason you wanted to relocate after the contrib section? In general, we should let towncrier order the sections (the order corresponds to the order in the config).
contrib/packaging/misc are placed at the very end since their audiences are small and most people won't need to read them.

Reply:
I had manual merged the notes together.  I checked them all back out into https://github.com/aio-libs/aiohttp/pull/8552 and will let towncrier do it